src/ToricVarieties/AlgebraicCycles/attributes.jl

```diff
@@ -0,0 +1,263 @@
+###########################################################################
+# 1. Defining attributes of rational equivalence class of algebraic cycles
+###########################################################################
+
+
+@doc Markdown.doc"""
+    toric_variety(ac::RationalEquivalenceClass)
+
+Return the normal toric variety of a rational
+equivalence class of algebraic cycles.
+
+# Examples
+```jldoctest
+julia> dP2 = del_pezzo(2)
+A normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+
+julia> d = ToricDivisor(dP2, [1,2,3,4,5])
+A torus-invariant, non-prime divisor on a normal toric variety
+
+julia> ac = RationalEquivalenceClass(d)
+A rational equivalence class on a normal toric variety represented by 6V(x3)+V(e1)+7V(e2)
+
+julia> toric_variety(ac)
+A normal, non-affine, smooth, projective, gorenstein, fano, 2-dimensional toric variety without torusfactor
+```
+"""
+toric_variety(ac::RationalEquivalenceClass) = ac.v
+export toric_variety
+
+
+@doc Markdown.doc"""
+    polynomial(ac::RationalEquivalenceClass)
+
+On a simplicial and complete toric variety, the Chow ring
+is isomorphic to a certain quotient of the Cox ring. This
+function returns the ring element corresponding to a given
+rational equivalence class of algebraic cycles.
```

Comment:
What is the difference between `representative` and `polynomial`? Is it only the ring?